### ISSUE TITLE
Support for ELFCLASS.ELFCLASS64 objects

### ISSUE THREAD

**Issue by rbertran** [v3l0c1r4pt0r/makeelf#16]
It seems that support for ELFCLASS64 objects is not yet implemented. I open this issue to see if there is some efforts going in that direction for coordination. 

Thanks,

**Comment by v3l0c1r4pt0r** [comment]
Yes, that's correct. AFAIK it is quite a big topic, so probably I will not be able to implement it myself. So, any help is welcome in this regards as well as any other missing part of ELF specifications.
